src/dev/build/tasks/bundle_fleet_packages.ts

```diff
@@ -43,15 +39,30 @@ export const BundleFleetPackages: Task = {
     const configFilePath = config.resolveFromRepo('fleet_packages.json');
     const fleetPackages = (await read(configFilePath)) || '[]';
 
+    const parsedFleetPackages: FleetPackage[] = JSON5.parse(fleetPackages);
+
+    log.debug(
+      `Found configured bundled packages: ${parsedFleetPackages
+        .map((fleetPackage) => `${fleetPackage.name}-${fleetPackage.version}`)
+        .join(', ')}`
+    );
+
     await Promise.all(
-      JSON5.parse(fleetPackages).map(async (fleetPackage: FleetPackage) => {
+      parsedFleetPackages.map(async (fleetPackage) => {
         const archivePath = `${fleetPackage.name}-${fleetPackage.version}.zip`;
         const archiveUrl = `${eprUrl}/epr/${fleetPackage.name}/${fleetPackage.name}-${fleetPackage.version}.zip`;
 
         const destination = build.resolvePath(BUNDLED_PACKAGES_DIR, archivePath);
 
         try {
-          await downloadPackageArchive({ log, url: archiveUrl, destination });
+          await downloadToDisk({
+            log,
+            url: archiveUrl,
+            destination,
+            shaChecksum: '',
+            shaAlgorithm: 'sha512',
+            skipChecksumCheck: true,
```

Comment:
Is it a possibility to get checksums enabled for these packages?

Reply:
It's very challenging because any time the package promotion job runs to update the package in a given EPR environment, every `.zip` for every package is recreated resulting in invalidated checksums for these archive files. See some of our findings here: https://github.com/elastic/kibana/issues/124273

Reply:
Looks like they're working on providing checksums for these artifacts in https://github.com/elastic/package-storage/issues/2776, we can readdress this once that's complete. Thanks!